nix/migration-tests.nix

```diff
@@ -112,6 +112,7 @@ let
           migrationTest
           pkgs.bash
           pkgs.coreutils
+          pkgs.python3
```

Comment:
Will pulling this in lead to an increase in build times for CI? (Or is everything cached?)

Reply:
It's all cached, and python3 is already used by openapi-spec-validator anyway.

Reply:
Okay, thanks for confirming!